episodes/map-and-reduce.md

```diff
@@ -78,9 +77,9 @@ bag.map(f).visualize()
 
 ### Filter
 
-To illustrate the concept of `filter`, it is useful to have a function that returns a boolean.
-In this case, we'll use a function that returns `True` if the argument contains the letter 'a',
-and `False` if it doesn't.
+A function returning a boolean is a useful illustration of the concept of `filter`.
```

Comment:
Now this sentence is factually wrong. It needs to say:
To illustrate the concept of `filter` we need a predicate, which is a function returning either `True` or `False`.

Reply:
Edited.